addons/website_sale/views/templates.xml

```diff
@@ -480,8 +480,8 @@
                             <p t-elif="not product.active" class="alert alert-warning">This product is no longer available.</p>
                             <p t-else="" class="alert alert-warning">This product has no valid combination.</p>
                             <hr t-if="product.description_sale" />
-                            <div class="o_not_editable">
-                                <p t-field="product.description_sale" class="text-muted" />
+                            <div>
```

Comment:
By removing `o_not_editable`, the `product_attributes_simple`  a few lines below also became editable which it should not, please add `product_attributes_simple` to it.

Reply:
done.!